### DISCUSSION TITLE
Move into the Vaultwarden organization

### DISCUSSION THREAD

**Discussion by Neustradamus** [dani-garcia/vaultwarden#5170] [Ideas]
Dear @vaultwarden team, @dani-garcia,

Currently there is a problem, an historical problem!

It will be better to tranfer https://github.com/dani-garcia/vaultwarden into https://github.com/vaultwarden.

Doc:
- https://help.github.com/en/articles/transferring-a-repository

Do you want help?

I have already done with success!

Thanks in advance.

Linked to:
- https://github.com/dani-garcia/vaultwarden/issues/5169

--------------

EDIT:

Solution: There is this PR:
- https://github.com/dani-garcia/vaultwarden/pull/4589

**Comment by AnonimousX1** [comment]
explain ?

**Reply by Neustradamus** [reply]
@AnonimousX1:

I have answered you here (2024-11-12):
- https://github.com/dani-garcia/vaultwarden/discussions/5170#discussioncomment-11221327

I have answered you here (2024-11-15):
- https://github.com/dani-garcia/vaultwarden/discussions/5170#discussioncomment-11261617

I have answered you here (2025-01-08): Happy New Year 2025 too!
- https://github.com/dani-garcia/vaultwarden/discussions/5170#discussioncomment-11771433

**Comment by Neustradamus** [comment]
@AnonimousX1: It is currently in a personal account instead of vaultwarden organization.

**Reply by stefan0xC** [reply]
That's not an explanation why it would be "a problem, an historical problem!".

**Reply by AnonimousX1** [reply]
Exactly. I don't understand what this historical problem is. Can you give more references and talk more about this?

**Reply by larena1** [reply]
https://github.com/mintty/mintty/issues/1297

It seems he is some kind of troll when it comes to that posting dozens of these requests and annoying people

**Reply by Neustradamus** [reply]
@AnonimousX1, @stefan0xC: If it is not clear, I will explain again: there is an organization for Vaultwarden "@vaultwarden", it is very good but the main repository is not into -> it is an historical problem.
When people are here: https://github.com/vaultwarden, they do not see the repository, it does not contribute, etc.
Vaultwarden is a good project :)

When @dani-garcia will do transfer, I hope soon, there will be no lost, GitHub adds all redirections, all links, all issues, all discussions, all PRs are redirected perfecly, all stars, all forks, all watchers, all vulnerabilities too.

**Reply by stefan0xC** [reply]
We are aware. But it's not a problem. And your arguments why make no sense. https://github.com/dani-garcia/vaultwarden/ is the canonical URL for the repository that is also linked in web-vault.

btw: In case you missed it there's also already an PR to change this: https://github.com/dani-garcia/vaultwarden/pull/4589

**Reply by Neustradamus** [reply]
@stefan0xC: Thanks a lot for this biggest information :)
I have commented in the specified PR!

**Comment by Neustradamus** [comment]
Dear all, @dani-garcia has created the @vaultwarden organization (good job!), it is easy to transfer the repository and others into.

When you will have done it, there will have redirections https://github.com/dani-garcia/vaultwarden/* to https://github.com/vaultwarden/vaultwarden/*, no lost.

Examples:
- https://github.com/nerzhul/ocsms -> https://github.com/nextcloud/ocsms
- https://github.com/tigaseinc/tigase-server -> https://github.com/tigase/tigase-server
- https://github.com/boothj5/profanity -> https://github.com/profanity-im/profanity
- https://github.com/asciimoo/searx -> https://github.com/searx/searx

Of course several projects are not good yet, it is in progress in the World, to respect, personal accounts and projects.

**Reply by BlackDex** [reply]
I'm not really sure what this comment adds to this discussion.
Redirection will be done by GitHub it self so nothing to worry about. And we will move it when we move it.

**Reply by Neustradamus** [reply]
@BlackDex: @stefan0xC has requested me to add an answer here, only @dani-garcia can do this step, and it is never too late to do it :)
It is currently not logic to have the good organization and one repository in this personal account.
When people are here -> https://github.com/vaultwarden, they do not find the repository and can not contribute, etc.

**Reply by stefan0xC** [reply]
That's not what I requested. I only pointed out that the answer "we will move it when we move it" was already given.

**Comment by Neustradamus** [comment]
Dear all, @vaultwarden team, @dani-garcia, I wish you a Happy New Year 2025!

Have you progressed on this ticket?

**Reply by larena1** [reply]
Hi @Neustradamus, I think there are still some questions here that you didn't answer. Would you mind doing that?

**Reply by Neustradamus** [reply]
@larena1: Thanks for your answer, where, I have not answered?

**Reply by larena1** [reply]
Please thoroughly go through this topic here again

**Reply by Neustradamus** [reply]
@larena1: I have answered to all, I do not see, sorry.

**Reply by stefan0xC** [reply]
And @BlackDex answered you here: https://github.com/dani-garcia/vaultwarden/discussions/5170#discussioncomment-11263449

**Reply by larena1** [reply]
https://github.com/dani-garcia/vaultwarden/discussions/5170#discussioncomment-11221327

Did you see the replies there? People weren't really satisfied with your previous answers unfortunately

**Reply by Neustradamus** [reply]
@stefan0xC: I have done new answers, I think it is now clear for you.

**Comment by dani-garcia** [comment]
I'll be closing this discussion as it doesn't seem like it is being very productive.

As it's been mentioned a few times the move to an organization has been in the plans for a while, but it hasn't been a priority as we've had other more important things to deal with. The move will happen but we can't promise when that will be.

If you want to be informed about it, the best way would be to subscribe to the linked PR (https://github.com/dani-garcia/vaultwarden/pull/4589). Once that PR is merged the repo will be moved into the organization.

Please don't repeatedly ask for updates because the answer will always be the same: It will be done when it's done.

**Reply by Neustradamus** [reply]
@dani-garcia: Thanks for your comment, I have discovered the PR in a @stefan0xC answer.
- https://github.com/dani-garcia/vaultwarden/pull/4589

I am happy to see that it is really taken in consideration :)
